### ISSUE TITLE
Emoji string fails to decode

### ISSUE THREAD

**Issue by bbhtt** [mccutchen/go-httpbin#152]
Hello any idea why this string fails to decode?

```
🙋, 🌍!\n…
```

It seems to work fine on command line `echo "8J+Ziywg8J+MjSEK4oCm"|base64 --decode` but https://httpbingo.org/base64/decode/8J+Ziywg8J+MjSEK4oCm returns an error.

**Comment by mccutchen** [comment]
The problem here is that go-httpbin only accepts the URL-safe base64 encoding, though the error does not make that obvious.

Fixing in https://github.com/mccutchen/go-httpbin/pull/153.

**Comment by mccutchen** [comment]
Fix released in [v2.12.0](https://github.com/mccutchen/go-httpbin/releases/tag/v2.12.0), and deployed to https://httpbingo.org.

Your [test case](https://httpbingo.org/base64/decode/8J+Ziywg8J+MjSEK4oCm) now works as expected:

```bash
$ curl 'https://httpbingo.org/base64/decode/8J+Ziywg8J+MjSEK4oCm'
🙋, 🌍!
…
```

Thanks for the report @bbhtt!

**Comment by bbhtt** [comment]
Thanks!
